oaknational/Oak-Web-Application
# I feat/iwant to show duplicate lessons and units in drop down LESQ-462

## Description

If a lesson or unit has duplicate results (same unit, different paths), then don’t display all those duplicates, instead hide them within a drop down on a card that contains the information of ALL The units/lessons 

https://www.notion.so/oaknationalacademy/I-want-owa-to-be-able-to-handle-duplicate-lessons-and-units-within-a-drop-down-b9fdf12bd68d4d59868d7da9779350fb

Music year: 2011

- Adds pathway to legacy and new API
- Adds dropdown to search hit card when there is more than one pathway in a search hit  - _add &flag=enable_2023_search_api to see results from new search api_
- updates tests


## How to test

1. Go to https://deploy-preview-2131--oak-web-application.netlify.thenational.academy
Search should work as before, you should see drop down for search hits with more than one pathway. Waves is a good one to test.


## Checklist
